internal/services/azure_reservation_service.go

```diff
@@ -147,9 +149,10 @@ func CreateAzureReservation(w http.ResponseWriter, r *http.Request) {
 		EdgeID:    logging.EdgeRequestId(r.Context()),
 		AccountID: identity.AccountId(r.Context()),
 		Args: jobs.LaunchInstanceAzureTaskArgs{
+			// Ignore location for now
+			//Location:          reservation.Detail.Location,
```

Comment:
We have ignored location previously and always substituted it for `eastus`, now we're just explicit about it.